### DISCUSSION TITLE
Customize action icon placement for collapsible gittree

### DISCUSSION THREAD

**Discussion by bigfooted** [Kitware/trame#251] [Q&A] [answered]
Hi all,

I was playing with the advanced gittree example (https://github.com/Kitware/trame/tree/master/examples/04_trame-components), that has collapsible trees. I would like to move the collapse/expand icon such that it is left-aligned, right next to the node text, or maybe even left of the the node/node text like many regular treeviews do. How can that be done?

**Comment by jourdain** [comment, marked as answer]
Looking at [that line](https://github.com/Kitware/trame-components/blob/master/vue-components/src/components/GitTree/script.js#L144) is seems we only expect them to be right aligned. 

But the web component could be extended to add such capability.

**Reply by bigfooted** [reply]
Thanks for the answer!
